include/zephyr/arch/arm/aarch32/cortex_a_r/scripts/linker.ld

```diff
@@ -21,10 +21,10 @@
 /* physical address of RAM */
 #ifdef CONFIG_XIP
   #define ROMABLE_REGION FLASH
-  #define RAMABLE_REGION SRAM
+  #define RAMABLE_REGION RAM
 #else
-  #define ROMABLE_REGION SRAM
-  #define RAMABLE_REGION SRAM
+  #define ROMABLE_REGION RAM
+  #define RAMABLE_REGION RAM
```

Comment:
As they both have `#define RAMABLE_REGION RAM`, could we put `#define RAMABLE_REGION RAM` out of the #ifdef preprocessor?

Reply:
Done.